src/services/clipService.ts

```diff
@@ -122,10 +147,13 @@ export async function getClipTextModelPath() {
             throw Error(CustomErrors.MODEL_DOWNLOAD_PENDING);
         } else {
             const localFileSize = (await fs.stat(modelSavePath)).size;
-            if (localFileSize !== TEXT_MODEL_SIZE_IN_BYTES) {
-                log.info('clip model size mismatch, downloading again');
+            if (localFileSize !== TEXT_MODEL_SIZE_IN_BYTES[type]) {
+                log.info(
+                    'clip text model size mismatch, downloading again',
```

Comment:
Make it consistent as the earlier log? `clip text model size mismatch, downloading again got:`